scripts/install

```diff
@@ -48,6 +49,7 @@ ynh_app_setting_set --app=$app --key=domain --value=$domain
 ynh_app_setting_set --app=$app --key=path --value=$path_url
 ynh_app_setting_set --app=$app --key=admin --value=$admin
 ynh_app_setting_set --app=$app --key=user_home --value=$user_home
+ynh_app_setting_set --app=$app --key=phpversion --value=$phpversion
```

Comment:
No need to do that:

https://github.com/YunoHost/yunohost/blob/dev/data/helpers.d/php#L84
https://github.com/YunoHost/yunohost/blob/d1ab1f674eb25f0970ded4698a89aee18f202cec/data/helpers.d/php#L120

Reply:
> and maybe wait a bit to get a nextcloud 23.0.1?

I have clearly no clues on how to setup config-panels... 😅

Reply:
I will try to spend some time on it :+1: